### DISCUSSION TITLE
What is the process to edit/update URL for an existing repository (Helm)

### DISCUSSION THREAD

**Discussion by bczoma** [artifacthub/hub#2997] [General]
We have Helm charts published from `https://COMPANY.github.io/`**REPO-ABC**`/helm-charts/` and it works, our Helm charts are accessible from Artifact Hub.

We need to rename the repo and publish from the new name: `https://COMPANY.github.io/`**REPO-XYZ**`/helm-charts/`
We cannot rely on GitHub forwarding old repo to the new repo name as we'll reuse the old name for other purposes.

Can someone please advise what the process is to initiate this change in Artifact Hub?

Tried through https://artifacthub.io Control Panel but there is only an option to Add a new Helm chart.

I haven't claimed repo ownership, maybe editing an existing chart gets enabled with that? 

Thanks in advance!

**Comment by tegioz** [comment]
Hi @bczoma 👋 

You can do it from the control panel (edit option):

<img width="232" alt="Screenshot 2023-04-21 at 18 20 17" src="https://user-images.githubusercontent.com/1213902/233686247-82bac8c2-ca97-486c-9319-985d41ce698e.png">

**Comment by bczoma** [comment]
Thanks for the quick answer @tegioz  - so the option exists, that's great. Looks like I need to claim ownership first, correct?

**Comment by tegioz** [comment]
No worries! Yes, if you don't have access to it from the control panel yet you'll need to claim its ownership.

**Comment by bczoma** [comment]
Got the answer, closing
